arches_lingo/src/arches_lingo/types.ts

```diff
@@ -46,13 +46,23 @@ export interface ControlledListeItemLabelValue {
 }
 
 export interface ControlledListItem {
+    item_id?: string;
     list_id: string;
     uri: string;
-    sortorder: number;
-    guide: boolean;
+    sortorder?: number;
+    guide?: boolean;
```

Comment:
Looks like this is morphing into the referencedatatype tile shape instead of the ControlledListItem django model. If that's right, how about we remove these props altogether and rename the type?